test/NuGet.Core.Tests/NuGet.Protocol.Core.v3.Tests/project.json

```diff
@@ -25,7 +25,7 @@
       "dependencies": {
         "moq.netcore": "4.4.0-beta8",
         "dotnet-test-xunit": "1.0.0-rc2-173361-36",
-        "System.Diagnostics.TraceSource": "4.0.0-rc3-24210-10",
+        "System.Diagnostics.TraceSource": "4.0.0",
         "Microsoft.NETCore.App": {
           "type": "platform",
           "version": "1.0.0-rc3-004338"

```

Comment:
> No, we can't update test until we get a new CLI

What is blocking you here?  You can use any version of `Microsoft.NETCore.App` without updating your CLI.